_config.yml

```diff
@@ -1,34 +1,38 @@
 # Permalinks
 #
 # Use of `relative_permalinks` ensures post links from the index work properly.
-permalink:           pretty
+permalink: pretty
 
 # Setup
-title:               Operação Serenata de Amor
-tagline:             Robôs contra a corrupção
-url:                 https://serenatadeamor.org
-baseurl:             ""
+title: Operação Serenata de Amor
+tagline: Inteligência artificial para controle social da administração pública
+url: https://www.serenatadeamor.org
+baseurl: ""
 
 # Assets
 #
 # We specify the directory for Jekyll so we can use @imports.
 sass:
-  sass_dir:          _sass
-  style:            :compressed
+  sass_dir: _sass
+  style: :compressed
 
 # About/contact
 author:
-  name:              Data Science Brigade
-  url:               https://www.facebook.com/operacaoSerenataDeAmor
-  email:             contato@datasciencebr.com
+  name: Operação Serenata de Amor
+  url: https://www.serenatadeamor.org
```

Comment:
What about synchronizing and release in `https://serenata.ai` instead?

Reply:
@cuducos serenata.ai is not working yet, so let's use .org from now.

Reply:
> serenata.ai is not working yet, so let's use .org from now.

What do you mean by _is not working_ yet? `jarbas.serenata.ai` is working. I think it's worse to have stuff in both. As we started to migrate, we should either continue to migrate or rollback whatever was migrated.

I'll stay with your _now_, but if you disagree with `serenata.ai` please let's talk about it.

Reply:
jarbas.serenata.ai is working, not serenata.ai.